humanmade/coding-standards
# Add Lerna and update project meta

Fixes #115 

### Outline

- This pull request seeks to add [Lerna](https://lerna.js.org/) for publishing packages to _npm_

- This pull request also updates and standardizes existing project meta.

- To aid in reviewing this pull request use the [hide-whitespace-change](https://github.com/humanmade/coding-standards/pull/175/files?utf8=%E2%9C%93&diff=unified&w=1) view

### Testing

1. Checkout this pull request locally and run `npm install`
1. Run `npm run publish:check`

<img width="796" alt="image" src="https://user-images.githubusercontent.com/1016458/72217464-2326fe00-3582-11ea-913f-2f92efdad6b0.png">
